mass_mailing_sending_queue/models/mail_mass_mailing.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from copy import deepcopy
+from openerp import api, fields, models, _
+from openerp.exceptions import Warning as UserError
+import logging
+
+
+class MailMassMailing(models.Model):
+    _inherit = 'mail.mass_mailing'
+
+    state = fields.Selection(selection_add=[
+        ('sending', "Sending"),
+    ])
+    pending = fields.Integer(string="Pending", compute='_compute_pending')
```

Comment:
This seems to be more a boolean than an integer. Use the suffix `_count` for noting the number